docs: add configure connector (en-US) (#97)
logto-io/docs: docs/docs/recipes/configure-connectors/README.md

---

# 🪛 Configure connectors

_Connectors_ are the bridge between Logto and other third-party vendors who provide short message service (SMS), email service, or user information on wildly accepted social media.

Logto Team provides you with many connectors to give end-users access to third-party providers' services. Such as Google, GitHub, Facebook, SendGrid mail, etc.

> You may feel annoyed when using such robust connectors:sob:?<br/>

Author: Tried to build a comparison between the abilities of connectors and the loss of developers when first see this conception.
Rephrased in another way.

Don't worry, this doc will show you an easy way to have things done!

## :watch: Kick-off :zap:

> Let's start from [Admin Console](https://logto.dev/console), signing in with your Logto account ("admin" access needed). You may [create an admin account](../../tutorials/get-started/#create-an-admin-account) if you don't have one.

Reviewer: should use their own env
```suggestion
Let's start from Admin Console, signing in with your Logto account ("admin" access needed). You may [create an admin account](../../tutorials/get-started.mdx#create-an-admin-account) if you don't have one.
```

Click the **Connectors** button on the sidebar :neutral_face:

> :ghost:**Welcome to the world of _connectors_**:ghost:

Reviewer: ```suggestion
:ghost: **Welcome to the world of _connectors_** :ghost:
```

---

# 🪛 Configure connectors

_Connectors_ are the bridge between Logto and other third-party vendors who provide short message service (SMS), email service, or user information on wildly accepted social media.

Logto provides you with many connectors to give end-users access to third-party providers. Such as Google, GitHub, Facebook, SendGrid mail, etc.

> You may feel lost when using connectors as this is a brand new concept :sob:

Reviewer: ```suggestion
You may feel lost when using connectors as this is a brand new concept. :sob:
```

> Don't worry; this doc will show you an easy way to have things done!

Reviewer: ```suggestion
Don't worry. This doc will show you an easy way to have things done!
```

## :watch: Kick-off :zap:

> Let's start from Admin Console, signing in with your Logto account ("admin" access needed). You may [create an admin account](../../tutorials/get-started/README.mdx#create-an-admin-account) if you don't have one.

Reviewer: ```suggestion
Let's start from Admin Console, signing in with your Logto account ("admin" role needed). You may [create an admin account](../../tutorials/get-started/README.mdx#create-an-admin-account) if you don't have one.
```

Click the "Connectors" button on the sidebar :neutral_face:

:ghost: **Welcome to the world of _connectors_** :ghost:

_Connectors_ can be classified into three types: _SMS Connector_, _Email Connector_ and _Social Connector_. See [connector reference](../../references/connectors/README.md) for more details about connector type.

You can continue setting up _Email connector_ or _SMS connector_, and you need to go to "Social connectors" tab before adding _Social connector_.

:::note
Move on to detailed guides for different connector types' configuration: [_SMS or Email connector_](./sms-or-email-connector.md) or [_Social connector_](./social-connector.md).

> :::

Reviewer: optional: I think these two pages are too short. maybe just combine them in 1 page?
